src/main/scala/firrtl/stage/Forms.scala

```diff
@@ -84,7 +84,8 @@ object Forms {
       Dependency(passes.PadWidths),
       Dependency(passes.memlib.VerilogMemDelays),
       Dependency(passes.SplitExpressions),
-      Dependency[firrtl.transforms.LegalizeAndReductionsTransform]
+      Dependency[firrtl.transforms.LegalizeAndReductionsTransform],
+      Dependency[firrtl.transforms.DedupAnnotationsTransform]
```

Comment:
I think this should actually be part of `ChirrtlForm` and `EliminateTargetPaths` invalidates it. Also, any transform which implements the `ResolvedAnnotationPaths` trait would now invalidate annotation deduplication.

Does that make sense to you?

Reply:
Yes, that does make sense. I noticed that the actually deduplication passes themselves were in `HighForm`, but I was previously pointed to `VerilogMinimumOptimized` being where I would put this pass. Clearly that approach ends up breaking some tests, along with `LowFormMinimumOptimized`, so I'll try and get some more clarification on where it should belong.

Reply:
Also, any change to `Forms.scala` will cause the `LoweringCompilersSpec` to break. That is a super-legacy thing that is hard-coding the transform order _before_ the Dependency API was added. That test needs to be updated if anything in `Forms.scala` changes or if dependencies change. 

(It's expected that that will break with the change you made here! That will need a patch.)

Reply:
It seems that adding `DedupAnnotationsTransform` to `Deduped` which is then appended to `HighForm` does not require a change to `LoweringCompilersSpec`